packages/paste-icons/tools/actions/convertAllAction.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
+const path = require('path');
+const {convertNewAction} = require('./convertNewAction');
+const {listIconsAction} = require('./listIconsAction');
+const {REACT_PATH} = require('../constants');
+
+function clearBuiltFiles() {
+  fs.readdir(REACT_PATH, (readErr, files) => {
+    if (readErr) throw readErr;
+
+    files
+      .filter(fileName => fileName.includes('.tsx')) // skips the helpers folder
+      .forEach(fileName => {
+        fs.unlink(path.join(REACT_PATH, fileName), unlinkErr => {
+          if (unlinkErr) {
+            throw unlinkErr;
+          }
+        });
+      });
+  });
+}
+
+async function convertAllAction() {
+  clearBuiltFiles();
+  await convertNewAction();
+  // filesystem operations take time, giving a delay to the next step so it doesn't fail
+  setTimeout(listIconsAction, 2000);
```

Comment:
This feels like it might be a little fragile. My async await knowledge it literally zero, don't awaits chain?

```
async function convertAllAction() {
  clearBuiltFiles();
  await convertNewAction();
  await listIconsAction();
}
```

Reply:
It is fragile :(, that was my first attempt.  It looks like some part of svgr or svg-to-react isn't respecting await.  I'll dig into it.

Reply:
As per Slack, let's stick a pin in this for now. It's good enough